helm/flottbot/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.3.0"
```

Comment:
should we get rid of this? it looks like it will pull `latest`. would like to avoid having to update this with new releases.

Reply:
that's basically just Helm Chart Metadata so yeah you I'll just rm it. the actual image tag is set here https://github.com/target/flottbot/pull/119/files#diff-5134e2967c1ad5dec02490146f55c302R9

Reply:
updated